platform-sdk/swirlds-merkledb/src/main/java/com/swirlds/merkledb/collections/LongListDisk.java

```diff
@@ -57,6 +57,10 @@ public class LongListDisk extends AbstractLongList<Long> {
      */
     private final FileChannel currentFileChannel;
 
+    /** This file channel is to work with the temporary file which is initialized in {@link LongListDisk#readBodyFromFileChannelOnInit}.
+     */
+    private FileChannel tempFileChannel;
```

Comment:
Would `initFileChannel` be a better name? This channel is only used when the list is initialized from a file on disk.